### PR TITLE
Add API unit tests

### PR DESCRIPTION
**Pull request by carrieedwards**
This PR adds unit tests for the api.go file.

### CODE REVIEW

**Review by leizor — DISMISSED** [review]
😎

File: pkg/influx/api_test.go [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+						},
+					},
+				}).Return(nil)
+				return remoteWriteMock
+			},
+			recorderMock: func() *MockRecorder {
+				recorderMock := &MockRecorder{}
+				recorderMock.On("measureMetricsParsed", 1).Return(nil)
+				recorderMock.On("measureMetricsWritten", 1).Return(nil)
+				recorderMock.On("measureConversionDuration", mock.Anything).Return(nil)
```
Comment (ywwg) [inline pkg/influx/api_test.go:55]: can you confirm that the duration is at least not zero?

**Review by ywwg — APPROVED** [review]
nice! just one question about the duration checks but that's it

File: pkg/influx/api_test.go (hunk 1 above)
Comment (carrieedwards) [reply pkg/influx/api_test.go]: Used mock.MatchedBy to make sure the number is a positive number when expected

**Pull request merged**